packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -3497,6 +3497,28 @@ describe('plugin-meetings', () => {
         });
       });
 
+      describe('#postEvent', () => {
+        it('should have #postEvent', () => {
+          assert.exists(meeting.postEvent);
+        });
+
+        it('should trigger `postEvent`', async () => {
+          await meeting.postEvent(eventType.LEAVE);
+          assert.calledWithMatch(Metrics.postEvent, {event: eventType.LEAVE});
+        });
+      });
+
+      describe('#setCaCorrelationId', () => {
+        it('should have #postEvent', () => {
```

Comment:
test title is incorrect

Reply:
this test case should be removed by your suggestion.